src/fm/cp_fm_cholesky.F

```diff
@@ -75,26 +89,36 @@ SUBROUTINE cp_fm_cholesky_decompose(matrix, n, info_out)
 #if defined(__parallel)
       desca(:) = matrix%matrix_struct%descriptor(:)
 #if defined(__DLAF)
-      IF (matrix%use_sp) THEN
-         CALL cp_pspotrf_dlaf('U', my_n, a_sp(:, :), 1, 1, desca, info)
-      ELSE
-         CALL cp_pdpotrf_dlaf('U', my_n, a(:, :), 1, 1, desca, info)
-      END IF
-#else
-      IF (matrix%use_sp) THEN
-         CALL pspotrf('U', my_n, a_sp(1, 1), 1, 1, desca, info)
+      WRITE(*,*) "DEBUG >>> dlaf_cholesky_n_min = ", dlaf_cholesky_n_min
```

Comment:
left over

Reply:
I thought I removed these, I'll double check the whole PR...